src/interceptor.ts

```diff
@@ -197,6 +204,10 @@ export const aws4Interceptor = <D = any>({
       }
     }
 
+    if (signingOptions.addContentSha) {
```

Comment:
It probably makes more sense to pass this via the [interceptor options](https://github.com/jamesmbourne/aws4-axios/blob/16aa61f56a3c981b52e746b42215353354dfd945/src/interceptor.ts#L17) and ignore `signingOptions` altogether, as this is only relevant to how `aws4` signs the request, which shouldn't be any different for this.